// Search `pornfilterlevel` CGI parameter representation
enum EPornoFilterMode {
    // Do not filter porno.
    PFM_OFF = 0;
    // Moderate porno filtration, this is the default level of filtration.
    PFM_MODERATE = 1;
    // Filter porno.
    PFM_STRICT = 2;
}
